lib/index.js

```diff
@@ -0,0 +1,255 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const base64url = require('base64url-universal');
+const bedrock = require('bedrock');
+const crypto = require('crypto');
+const database = require('bedrock-mongodb');
+const {promisify} = require('util');
+const uuid = require('uuid-random');
+const {util: {BedrockError}} = bedrock;
+
+// load config defaults
+require('./config');
+
+const AES_KW_ALGORITHM = 'id-aes256-wrap';
+const AES_KW_RFC3394_IV = new Buffer.from('A6A6A6A6A6A6A6A6', 'hex');
```

Comment:
`new` seems to work but [could be removed](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_string_encoding).
```suggestion
const AES_KW_RFC3394_IV = Buffer.from('A6A6A6A6A6A6A6A6', 'hex');
```